magenta/models/sketch_rnn/model.py

```diff
@@ -244,7 +244,9 @@ def build_model(self, hps):
     self.num_mixture = hps.num_mixture
 
     # TODO(deck): Better understand this comment.
```

Comment:
I'm not sure who deck is, but I hope he would find the new version of this comment more understandable. :)